app/models/chef.rb

```diff
@@ -1,4 +1,10 @@
 class Chef < ApplicationRecord
    validates_presence_of :name
    has_many :dishes
+   has_many :dish_ingredients, through: :dishes
+   has_many :ingredients, through: :dish_ingredients
```

Comment:
Awesome use of additional associations. Because dishes and ingredients also have an association set up between them in their models, you can actually consolidate these two lines like so: `has_many :ingredients, through: :dishes`

We use the joins object (dish_ingredients) in their models to set up their association between each other and because of that, we can operate as if they have a direct relationship when we refer to them in associations outside of their own models